### DISCUSSION TITLE
Ayuda a descargar los modulos <3

### DISCUSSION THREAD

**Discussion by GaelOlvera11** [LaunchX-InnovaccionVirtual/CursoIntroPython#291] [Q&A] [answered]
Ayuda quisiera descargar los archivos para verlos en mi visual studio:(

**Comment by OscarPedroza** [comment, marked as answer]
Hola, puedes descargar los archivos desde el repositorio en el botón verde de 'code' y se te desplegará una opción para poder descargarlo como un zip:

![image](https://user-images.githubusercontent.com/83786890/154406716-8465a496-ba56-4c4d-9ee7-5e4e5da51e7d.png)

**Reply by GaelOlvera11** [reply]
Gracias broo
